prusa/link/web/cameras.py

```diff
@@ -80,10 +80,18 @@ def default_camera_snap(req):
 def list_cameras(_):
     """List all configured cameras"""
     camera_configurator = app.daemon.prusa_link.camera_configurator
+    id_list = []
```

Comment:
Suggestion: `id_set = set()`

Reply:
Hi, this is very much rushed, so I appreciate you taking a look. Indeed, if I wasn't after the camera order, this would be so much faster. But for the API simplicity, this list is what ultimately gets shown to the user. I should probably mention that in the doc string of the function tho. Thank you